controller/space.go

```diff
@@ -200,7 +225,7 @@ func (c *SpaceController) Delete(ctx *app.DeleteSpaceContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, err)
 	}
-	c.resourceManager.DeleteResource(ctx, ctx.RequestData, auth.Resource{ResourceID: resourceID, PermissionID: permissionID, PolicyID: policyID})
+	err = c.resourceManager.DeleteSpace(ctx, ctx.RequestData, ctx.SpaceID.String())
```

Comment:
goasupport.ForwardContextRequestID(ctx) ?

Reply:
goasupport.ForwardContextRequestID(ctx) is used inside resourceManager.Create/DeleteSpace()